packages/css-data/bin/property-value-descriptions.ts

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable no-console, func-style */
+import fetch from "isomorphic-fetch";
```

Comment:
Please use undid

Reply:
Why not standard fetch? The api is known by most devs, for example if I were to install undici I would need learn how it works

Reply:
I had no idea we want to use undici, but I just had a quick glance, it seems to have massive perf improvements and has the standard fetch https://github.com/nodejs/undici#undicifetchinput-init-promise